AdobeStockImageAdminUi/Test/Mftf/Test/AdminAdobeStockSeeMoreSeriesFilterTest.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="AdminAdobeStockSeeMoreSeriesFilterTest">
+        <annotations>
+            <stories value="Admin User see more images from series filter"/>
+            <title value="Admin should be able to see more images from series filter"/>
+            <description value="Admin should be able to see more images from series filter"/>
+            <severity value="CRITICAL"/>
+        </annotations>
+        <before>
+            <actionGroup ref="LoginAsAdmin" stepKey="loginAsAdmin"/>
+            <actionGroup ref="AdminOpenMediaGalleryForPageNoEditorActionGroup" stepKey="openMediaGalleryForPage"/>
+            <actionGroup ref="AdminAdobeStockOpenPanelFromMediaGalleryActionGroup" stepKey="openAdobeStockPanel"/>
+        </before>
+        <after>
+            <actionGroup ref="resetAdminDataGridToDefaultView" stepKey="resetAdminDataGridToDefaultView"/>
+            <actionGroup ref="logout" stepKey="logout"/>
+        </after>
+        <grabAttributeFrom selector="{{AdobeStockSection.firstImageId}}" userInput="data-id"
+                           stepKey="getFilteredId"/>
+        <click selector="{{AdobeStockFilterSection.filters}}" stepKey="expandFilters"/>
+        <waitForPageLoad stepKey="waitForFiltersOpen"/>
+        <actionGroup ref="AssertIsNotVisibleAdobeStockFilterElementActionGroup" stepKey="checkSerieIdFilter">
+            <argument name="filterName" value="Serie Id"/>
+        </actionGroup>
+        <click selector="{{AdobeStockFilterSection.cancelFilters}}" stepKey="cancelFilters"/>
+        <actionGroup ref="AdminAdobeStockExpandImagePreviewActionGroup" stepKey="expandImagePreview"/>
+        <actionGroup ref="AdminAdobeStockApplySameSeriesFilter" stepKey="applySerieFilter"/>
```

Comment:
Action group name should have `ActionGroup` suffix, otherwise the exception is thrown during generation:

    In TestGenerator.php line 245:
                                                                               
    Could not find ActionGroup by ref "AdminAdobeStockApplySameSeriesFilterActionGroup"
    in Test "AdminAdobeStockSeeMoreSeriesFilterTest"                    
    AdobeStockImageAdminUi/Test/Mftf/Test/AdminAdobeStockSeeMoreSeriesFilterTest.xml